app/Console/Commands/Cloud/CloudDeleteUser.php

```
@@ -54,124 +55,141 @@ public function handle()
             return 1;
         }
 
-        $this->logAction("Starting user deletion process for: {$email}");
+        // Implement file lock to prevent concurrent deletions of the same user
+        $lockKey = "user_deletion_{$this->user->id}";
+        $lock = Cache::lock($lockKey, 600); // 10 minute lock
```

PHPMD (2.15.0)

Avoid using static access to class '\Illuminate\Support\Facades\Cache' in method 'handle'. (undefined)

(StaticAccess)